_docs/node-operator-requirements.md

```diff
@@ -98,15 +130,49 @@ oak-collator \
   --execution=wasm \
   --no-telemetry
 ```
+
 It's important that we keep the state-cache-size to 0 for now. It causes downstream node issues down the road which will require more overhead for the node operator.
 
-If you're running a different OS, please compile the binary first and follow the instructions in the OAK-blockchain [README](https://github.com/OAK-Foundation/OAK-blockchain#install-oak-blockchain)
+#### Option 3: Docker users
+If you're using a Linux box, you can simply run the following for Neumann:
+```
+docker run -d -p 30333:30333 -p 9944:9944 -p 9933:9933 -v neumann-data:/data oaknetwork/neumann:1.3.0 \
+  --name=YOUR_COLLATOR_NAME \
+  --base-path=PATH_TO_DATA_DIR \
+  --chain=neumann \
+  --node-key=NODE_KEY \
+  --collator \
+  --force-authoring \
+  --execution=wasm \
+  --state-cache-size=0 \
+  -- \
+  --chain=neumann-relay \
+  --execution=wasm \
+  --no-telemetry
+```
 
-### Step 6: Check that your node is in the Telemetry list and is connected to the network
-If you're successful in connecting to the network and sending your Telemetry data, you can see your node's name (`YOUR_COLLATOR_NAME`) on [this list](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d).
+And the following command for Turing:
+```
+docker run -d -p 30333:30333 -p 9944:9944 -p 9933:9933  -v turing-data:/data oaknetwork/turing:1.3.0 \
+  --name=YOUR_COLLATOR_NAME \
+  --base-path=PATH_TO_DATA_DIR \
+  --chain=turing \
+  --node-key=NODE_KEY \
+  --collator \
+  --force-authoring \
+  --execution=wasm \
+  --state-cache-size=0 \
+  -- \
+  --execution=wasm \
+  --no-telemetry
+```
 
-### Step 7: Sync your node
+### Step 4: Check that your node is in the Telemetry list and is connected to the network
+If you're successful in connecting to the network and sending your Telemetry data, you can see your node's name (`YOUR_COLLATOR_NAME`) on [this list](https://telemetry.polkadot.io/#list/0x42e75532d6809775cef4b9ca8e4bb49be2dc1e87c9ff1ba299e78481b5cb3047).
+
+### Step 5: Sync your node
 Please ensure that both the relay chain block number and parachain block number are up to the latest block number. The logs will indicate whether or not the nodes are fully synced.
+While you're blocks are syncing, monitor the initialization, especially for the first few lines to ensure that you are pointing to the correct network. If you run into any issues, head over to the [OAK Discord](https://discord.gg/7W9UDvsbwh) for help.
 
-### Step 8: Onboard your collator
-Once you're fully synced, you're ready to on-board as a collator. Proceed to this page to move forward and to start producing blocks. [Collator On-boarding](../collators)
+### Step 6: Onboard your collator
+Once you're fully synced, you're ready to on-board as a collator. Proceed to this page to move forward and to start producing blocks. [Collator On-boarding](./collators.md)
```

Comment:
The link for `./collators.md` is broken for me (the old one at `../collators` works)